frontend/public/components/RBAC/role.jsx

```diff
@@ -238,6 +244,25 @@ export const BindingsForRolePage = (props) => {
   );
 };
 
+const getBreadcrumbs = (model, kindObj, match) => {
+  const lastNamespace = getLastNamespace();
+  return [
+    {
+      name: i18next.t(model.labelPluralKey) || model.labelPlural,
```

Comment:
Sorry, I missed this, but I wouldn't rely on the any particular behavior when `i18next.t(undefined)` is called. I have a feeling that didn't cause a problem because it's defined for both of these models, but I'd guess we'd get at least a missing key error.

```suggestion
      name: model.labelPluralKey ? i18next.t(model.labelPluralKey) : model.labelPlural,
```

Reply:
fixed